src/components/home/index.js

```diff
@@ -34,12 +42,11 @@ const Home = () => {
         setIsOptionKey(false);
       }}
     >
-      <UserProfile />
       {dev && <SearchBox />}
       <h1 className={styles.heading}>Designers</h1>
       <Designers />
       <h1 className={styles.heading}>{`${BRAND_NAME} ${MEMBERS_TITLE}`}</h1>
-      {isSuperUserMode && (
+      {isSuperUser && (
```

Comment:
What is the significance of changing isSuperUserMode to isSuperUser?

Reply:
Earlier superUser had to click on a checkbox to enable superUserMode and on enabling we set `isSuperUserMode ` to true. Now we got rid of it so directly using `isSuperUser` instead of `isSuperUserMode `.
![image](https://user-images.githubusercontent.com/97341921/190174995-bd487995-4a7b-4d27-8dc3-e35158610d78.png)